Marlin/Configuration.h

```diff
@@ -1442,6 +1442,17 @@
   //#define BD_SENSOR_PROBE_NO_STOP // Probe bed without stopping at each probe point
 #endif
 
+/**
+ * BIQU MicroProbe
+ *
+ * A lightweight, solenoid-driven probe.
+ * For information about this sensor https://github.com/bigtreetech/MicroProbe
+ *
+ * Also requires: PROBE_ENABLE_DISABLE
```

Comment:
I wanted to streamline this more & automatically define `PROBE_ENABLE_DISABLE` when one of the MicroProbe options are enabled, but then it duplicated the `PROBE_ENABLE_DISABLE`/`PROBE_ENABLE_PIN` config block. I left it as a sanity check for now in order to open a PR and get this in sooner rather than later.

Reply:
Better to make the user enable config options, including `PROBE_ENABLE_DISABLE`, rather than enable it transparently. This is the preferred policy for all config options that are exposed to the user, otherwise they are "conditionals."